databricks/resource_databricks_secret_scope.go

```diff
@@ -26,6 +26,8 @@ func resourceSecretScope() *schema.Resource {
 				Default:  "users",
 			},
 			"backend_type": {
+				// TODO: add default DATABRICKS as backend type.
+				// TODO: add integration tests for AKV backend
```

Comment:
I do not believe you can actually make akv backed scopes via apis at the moment

Reply:
nope, i've even checked with devs